fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/logging/LogMessageKeys.java

```diff
@@ -134,12 +134,15 @@
     DOCUMENT("document"),
     SESSION_ID("session_id"),
     INDEXER_SESSION_ID("indexer_session_id"),
+    INDEXER_ID("indexer_id"),
     INDEX_STATE_PRECONDITION("index_state_precondition"),
     INITIAL_INDEX_STATE("initial_index_state"),
     SHOULD_BUILD_INDEX("do_build_index"),
     SHOULD_CLEAR_EXISTING_DATA("clear_existing_data"),
     SHOULD_MARK_READABLE("should_mark_readable"),
     RESULT("result"),
+    INDEXER_CURR_RETRY("indexer_curr_retry"),
+    INDEXER_MAX_RETRIES("indexer_max_retries"),
```

Comment:
Was it purposeful to continue using "retry" here instead of `CURR_ATTEMPT` and `MAX_ATTEMPTS` (to standardize with the `Runner`)?

Reply:
I'm using "retry" here to differentiate this pair from the pair in the runner. Also, the config of OnlineIndexer uses "retry" rather than "attempt".

There will be off-by-one problem either way we choose, so the many intent of this change is make the curr & max pairs match. This problem should be able to be solved thoroughly in the retry abstraction I'm prototyping on currently.

Reply:
I see. I'm...not sure it wouldn't be better to standardize here and fix the off-by-one error at least as it displays in the logs, but that could also wait for a separate PR, I suppose.